CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.16.0 (January 2021)
+- Native Android ShareSheet
+- Option to log HTTP requests and responses
+- Move sort menu from toolbar to files view
+- Improvements in thumbnails
+- Update background image with new ci images
```

Comment:
What does ci mean here? i'd set it as `Update background image`

Reply:
It means corporative images. But it is ok, I can update it.